mergin/client.py

```diff
@@ -298,6 +301,12 @@ def get_push_changes(self):
         :rtype: dict
         """
         changes = self.compare_file_sets(self.metadata['files'], self.inspect_files())
+        # do checkpoint to push changes from wal file to gpkg if new file
+        for file in changes['added']:
+            if ".gpkg" in file["path"]:
+                do_sqlite_checkpoint(self.fpath(file["path"]))
+                file["checksum"] = generate_checksum(self.fpath(file["path"]))
```

Comment:
missing update of file['size'] hence number of chunks in next step could be incorrect